tests/core/parameters_date_indexing/aad_rg.yaml

```diff
@@ -0,0 +1,121 @@
+description: Age d'annulation de la décote
+age_annulation_decote_en_fonction_date_naissance:
```

Comment:
Do the parameters of `aad_rg.yaml` and `trimtp_rg.yml` show other mechanisms than the `before_YYYY_MM_DD` / `after_YYYY_MM_DD` format? 🤔 If they don't, we could shorten all the (translated) examples to highlight what they are describing.

Reply:
Yes a translation is needed